### PR TITLE
Update CSV files for package deprecation

### PR DESCRIPTION
**Pull request by johnemmanuelm**

### CODE REVIEW

File: _data/releases/latest/js-packages.csv [hunk 1]
```diff
@@ -270,7 +270,7 @@
...
 "@azure/arm-storagecache","6.0.0","7.0.0-beta.1","Resource Management - Storage Cache","Storage","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storagecache/arm-storagecache","","","mgmt","true","","","01/20/2022","","","","","","","",""
-"@azure/arm-storageimportexport","2.1.0","","Resource Management - Storage Import/Export","Storage","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storageimportexport/arm-storageimportexport","","","mgmt","true","","02/03/2023","01/23/2022","","","","","","","",""
+"@azure/arm-storageimportexport","2.1.0","","Resource Management - Storage Import/Export","Storage","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storageimportexport/arm-storageimportexport","","","mgmt","true","","02/03/2023","01/23/2022","deprecated","02/28/2023","","resourcemanager/databox/armdatabox/","https://github.com/Azure/azure-sdk-for-go/tree/sdk/resourcemanager/databox/armdatabox/v1.0.0/sdk/resourcemanager/databox/armdatabox/","","",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:273]: The replacement package and guide isn't correct here. It looks like you copied from go.

**Review by weshaggard — CHANGES_REQUESTED** [review]
The replacement package and mitigation links need to be fixed.

**Review by mario-guerra — CHANGES_REQUESTED** [review]
Hi, for Python, .NET, JS, and Java, the 'Replace' field should just be the package name that shows up in the language package manager, not the sdk repo path.

**Review by mario-guerra — APPROVED** [review]

File: _data/releases/latest/js-packages.csv [hunk 2]
```diff
@@ -277,7 +277,7 @@
...
 "@azure/arm-storage","18.1.0","","Resource Management - Storage","Storage","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storage/arm-storage","","","mgmt","true","","12/19/2022","12/06/2021","","","","","","","",""
 "@azure/arm-storagecache","6.0.0","7.0.0-beta.1","Resource Management - Storage Cache","Storage","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storagecache/arm-storagecache","","","mgmt","true","","","01/20/2022","","","","","","","",""
-"@azure/arm-storageimportexport","2.1.0","","Resource Management - Storage Import/Export","Storage","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storageimportexport/arm-storageimportexport","","","mgmt","true","","02/03/2023","01/23/2022","","","","","","","",""
+"@azure/arm-storageimportexport","2.1.0","","Resource Management - Storage Import/Export","Storage","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storageimportexport/arm-storageimportexport","","","mgmt","true","","02/03/2023","01/23/2022","deprecated","04/10/2023","","@azure/arm-storageimportexport","https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/databox/arm-databox/README.md","","",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:280]: Isn't the replacement package the same as the original package here?

Comment (johnemmanuelm) [reply _data/releases/latest/js-packages.csv]: Fixed

**Review by weshaggard — APPROVED** [review]

**Pull request merged**